src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -378,6 +378,19 @@ object ICollection.SyncRoot
             }
         }
 
+        int ICollection.Count 
```

Comment:
What value is there in having them in netstandard 2.0? Should they be removed from netstandard 2.0?